addons/l10n_pe/i18n/l10n_pe.pot

```diff
@@ -0,0 +1,9611 @@
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_130305
+msgid "Uchumarca"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_040124
+msgid "Uchumayo"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_050410
+msgid "Uchuraccay"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_021016
+msgid "Uco"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_130306
+msgid "Ucuncha"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_120504
+msgid "Ulcumayo"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_210809
+msgid "Umachiri"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_100804
+msgid "Umari"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_211307
+msgid "Unicachi"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_061113
+msgid "Unión Agua Blanca"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_050708
+msgid "Upahuacho"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_040413
+msgid "Uraca"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_030607
+msgid "Uranmarca"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_160305
+msgid "Urarinas"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_081201
+msgid "Urcos"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_130813
+msgid "Urpay"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_081301
+#: model:res.city,name:l10n_pe.city_pe_0813
+msgid "Urubamba"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_210310
+msgid "Usicayos"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_130614
+msgid "Usquil"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_060311
+msgid "Utco"
+msgstr ""
+
+#. module: l10n_pe
+#: model:res.city,name:l10n_pe.city_pe_0107
+msgid "Utcubamba"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_061310
+msgid "Uticyacu"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_040412
+msgid "Uñon"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_010311
+msgid "Valera"
+msgstr ""
+
+#. module: l10n_pe
+#: model:l10n_pe.res.city.district,name:l10n_pe.district_pe_160606
+msgid "Vargas Guerra"
```

Comment:
Probably districts should not be translatable.

Reply:
Removing all translations that will not be translated like districts, first PR was following the logic of exporting the POT like in l10n_mx_edi
https://github.com/odoo/enterprise/blob/14.0/l10n_mx_edi/i18n/l10n_mx_edi.pot